src/components/fileName/fileName.tsx

```diff
@@ -0,0 +1,14 @@
+import { useExcalidrawActionManager } from "../App";
+import { actionSaveToActiveFile } from "../../actions";
+function FileName(props: any) {
+  const actionManager = useExcalidrawActionManager();
+  return (
+    <>
+      <span onClick={() => actionManager.executeAction(actionSaveToActiveFile)}>
+        <b>{props.name.slice(0, -11)}</b>
```

Comment:
let's use CSS ellipsis with a max width of 180px.

As for removing extension, the extension may also be composite (e.g. `.excalidraw.png` etc.), so it's safer to consider all the keys of the MIME_TYPES object as possible extensions and strip those:

https://github.com/excalidraw/excalidraw/blob/b57b3b573da874f7527589c10632ec936c1962d2/src/constants.ts#L142-L154